src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -403,55 +403,6 @@ protected override ImmutableArray<PendingBranch> Scan(ref bool badRegion)
             return pendingReturns;
         }
 
-        protected override ImmutableArray<PendingBranch> RemoveReturns()
```

Comment:
Do we still need a `virtual` method for `RemoveReturns`?

Reply:
Yes, `ControlFlowPass` overrides it.